cogboard-webapp/src/components/Widget/helpers.js

```diff
@@ -56,3 +59,11 @@ export const renderCardContent = (
     </StyledCardContent>
   );
 };
+
+const ExpandableContent = ({ type, status, content, expanded }) => {
+  return type !== 'TextWidget' ? (
```

Comment:
Im not sure about this complex ternary, it is extremely hard to read. How about simplifying this? Could be in some separate ticket if there is no time for that now.

Reply:
Ok, I will refactor it in a moment

Reply:
done